### PR TITLE
provider-compatibility.yml: Correct the directory where opensslwrap.sh is being run

### PR DESCRIPTION
**Pull request by t8m**

### CODE REVIEW

File: .github/workflows/provider-compatibility.yml [hunk 1]
```diff
@@ -13,9 +13,9 @@ name: Provider compatibility across versions
 # NOTE: if this is being run on pull_request, it will **not** use the pull
 #       request's branch.  It is hardcoded to use the master branch.
 #
-on: #[pull_request]
-  schedule:
-    - cron: '0 15 * * *'
+on: [pull_request]
+#  schedule:
+#    - cron: '0 15 * * *'
```
[the remark this answers is not in the document]
Comment (t8m) [reply .github/workflows/provider-compatibility.yml:18]: No, this is to be dropped.

File: test/recipes/25-test_verify.t [hunk 2]
```diff
@@ -369,6 +369,19 @@ SKIP: {
     skip "EC is not supported or FIPS is disabled", 7
         if disabled("ec") || $no_fips;
 
+    $ENV{OPENSSL_CONF} = $provconf;
```
Comment (paulidale) [inline test/recipes/25-test_verify.t:372]: Why relocate these?

**Review by paulidale — APPROVED** [review]
okay with urgent

File: test/recipes/25-test_verify.t (hunk 2 above)
Comment (t8m) [reply test/recipes/25-test_verify.t:372]: They are not dependent on the FIPS provider version. There was also a bug in that skip command below - only 3 testcases were marked as skipped but these 4 were skipped too.

**Comment by t8m** [comment]
ping for second review - we need this merged to check whether provider compat CI is passing

**Review by mattcaswell — APPROVED** [review]
Ok with urgent

**Comment by t8m** [comment]
Merged to the master branch. Thank you for the reviews.

**Pull request closed without merging**